### DISCUSSION TITLE
Code highlighting

### DISCUSSION THREAD

**Discussion by richban** [LekoArts/gatsby-themes#727] [Questions] [answered]
Is there a way to shadow the current code highlighting?

I would like to use https://www.gatsbyjs.com/plugins/gatsby-remark-highlight-code instead.

**Comment by LekoArts** [comment, marked as answer]
You can set `mdx: false` as an option and then use your own instance of `gatsby-plugin-mdx` where you can define that remark plugin. You'd probably also need to shadow the `gatsby-plugin-theme-ui/components.js` file to not convert `<pre>` tags to components with the `<Code />` component.

**Reply by richban** [reply]
That's exactly what I have done! Thanks a lot! ;)

**Reply by LekoArts** [reply]
Great! :)
